Python 3.12 upgrade broke titiler

I got a runtime error on a fresh **titiler** deploy:

```
[ERROR] Runtime.ImportModuleError: Unable to import module 'handler': libexpat.so.1: cannot open shared object file: No such file or directory
```

A quick search produced [this AWS issue](https://repost.aws/questions/QUKvip5FFLT3yM4VLwNRjYug/lambda-python-3-12-3-13-getting-error-message-libexpat-so-1-cannot-open-shared-object-file-no-such-file-or-directory-with-python-3-13) (which ironically also uses **titiler**) that indicates that `libexpat` _might_ not exist on `Amazon Linux 2023`, which Python 3.12 uses.

We might want to yank [v10.1.0](https://pypi.org/project/eoapi-cdk/10.1.0/) of this package from PyPI, as well?

--- a/lib/titiler-pgstac-api/runtime/Dockerfile
+++ b/lib/titiler-pgstac-api/runtime/Dockerfile
@@ -10,6 +10,10 @@ RUN dnf install -y gcc-c++ findutils
 COPY titiler-pgstac-api/runtime/requirements.txt requirements.txt
 RUN python -m pip install -r requirements.txt "mangum>=0.14,<0.15" -t /asset
 
+# copy libexpat.so.1 into a location included in LD_LIBRARY_PATH in the Lambda runtime environment
+# (/usr/lib64 is not available in the Lambda runtime, just the build environment)
+RUN cp /usr/lib64/libexpat.so.1 /asset/
+
 # Remove system dependencies
 RUN dnf remove -y gcc-c++
 

